addons/respawn/XEH_postInit.sqf

```diff
@@ -1,5 +1,4 @@
-// by commy2
 #include "script_component.hpp"
 
 ["rallypointMoved", {_this call FUNC(updateRallypoint)}] call EFUNC(common,addEventhandler);
-["playerChanged", {_this call FUNC(handlePlayerChanged)}] call EFUNC(common,addEventhandler);  // hide enemy rallypoint markers
+["playerChanged", {_this call FUNC(handlePlayerChanged)}] call EFUNC(common,addEventhandler); // hide enemy rallypoint markers
```

Comment:
This is something me and @jokoho48 are considering right now. You're right in that this is not a good way of doing it, but it will not copy the entire function into the Evenhandler.
The consideration is to enable the ACE eventhandler system to allow for referencing function names in addition.